<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="0" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-02T22:58:26 IST" name="Page Object Model" finished-at="2020-02-02T22:59:17 IST" duration-ms="50981">
    <groups>
    </groups>
    <test started-at="2020-02-02T22:58:26 IST" name="Login test" finished-at="2020-02-02T22:58:26 IST" duration-ms="95">
      <class name="com.ka.testCases.testCaseLogin">
        <test-method signature="TestCaseLogin(java.util.Hashtable)[pri:0, instance:com.ka.testCases.testCaseLogin@f9d87b]" started-at="2020-02-02T22:58:26 IST" name="TestCaseLogin" data-provider="dp" finished-at="2020-02-02T22:58:26 IST" duration-ms="22" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Anusuya@63, runmode=N, username=karthikbdvt6@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test case as the runmode is N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test case as the runmode is N
at com.ka.testCases.testCaseLogin.TestCaseLogin(testCaseLogin.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCaseLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ka.testCases.testCaseLogin@f9d87b]" started-at="2020-02-02T22:58:26 IST" name="tearDown" finished-at="2020-02-02T22:58:26 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="TestCaseLogin(java.util.Hashtable)[pri:0, instance:com.ka.testCases.testCaseLogin@f9d87b]" started-at="2020-02-02T22:58:26 IST" name="TestCaseLogin" data-provider="dp" finished-at="2020-02-02T22:58:26 IST" duration-ms="4" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=kasgfhejfke, runmode=N, username=kushalgambir6@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the Test case as the runmode is N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the Test case as the runmode is N
at com.ka.testCases.testCaseLogin.TestCaseLogin(testCaseLogin.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestCaseLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ka.testCases.testCaseLogin@f9d87b]" started-at="2020-02-02T22:58:26 IST" name="tearDown" finished-at="2020-02-02T22:58:26 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ka.testCases.testCaseLogin -->
    </test> <!-- Login test -->
    <test started-at="2020-02-02T22:58:26 IST" name="Search Flight Test" finished-at="2020-02-02T22:59:17 IST" duration-ms="50338">
      <class name="com.ka.testCases.flightSearchTest">
        <test-method signature="FlightSearchTest(java.util.Hashtable)[pri:0, instance:com.ka.testCases.flightSearchTest@2cf23c81]" started-at="2020-02-02T22:58:26 IST" name="FlightSearchTest" data-provider="dp" finished-at="2020-02-02T22:58:26 IST" duration-ms="3" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{returning=15-11-2020, departing=04-11-2020, runmode=N, to=Madrid, Spain (MAD-All Airports), from=Bengaluru, India (BLR-Kempegowda Intl.)}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test case as the runmode is N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test case as the runmode is N
at com.ka.testCases.flightSearchTest.FlightSearchTest(flightSearchTest.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightSearchTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ka.testCases.flightSearchTest@2cf23c81]" started-at="2020-02-02T22:58:26 IST" name="tearDown" finished-at="2020-02-02T22:58:26 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="FlightSearchTest(java.util.Hashtable)[pri:0, instance:com.ka.testCases.flightSearchTest@2cf23c81]" started-at="2020-02-02T22:58:26 IST" name="FlightSearchTest" data-provider="dp" finished-at="2020-02-02T22:59:16 IST" duration-ms="49542" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{returning=20-12-2020, departing=02-12-2020, runmode=Y, to=London, United Kingdom (LON-All Airports), from=Bengaluru, India (BLR-Kempegowda Intl.)}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightSearchTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ka.testCases.flightSearchTest@2cf23c81]" started-at="2020-02-02T22:59:16 IST" name="tearDown" finished-at="2020-02-02T22:59:17 IST" duration-ms="765" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ka.testCases.flightSearchTest -->
    </test> <!-- Search Flight Test -->
  </suite> <!-- Page Object Model -->
</testng-results>
